frontend/src/pages/WhatIsHealthEquity/News/NewsPreviewCard.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { NEWS_TAB_LINK, ReactRouterLinkButton } from "../../../utils/urlutils";
+import styles from "./News.module.scss";
+import AppbarLogo from "../../../assets/AppbarLogo.png";
+import parse from "html-react-parser";
+import { Article } from "../NewsTab";
+import { Box, Grid } from "@material-ui/core";
+import LazyLoad from "react-lazyload";
+
+export interface NewsPreviewCardProps {
+  article: Article;
+  arrow?: "prev" | "next";
+}
+
+export default function NewsPreviewCard(props: NewsPreviewCardProps) {
+  const { article } = props;
+
+  return (
+    <ReactRouterLinkButton
+      url={`${NEWS_TAB_LINK}/${article.slug}`}
+      className={styles.NewsPreviewHeaderText}
+    >
+      <Grid container wrap="nowrap" justify="space-evenly">
+        <Grid
+          item
+          xs={1}
+          container
+          direction="column"
+          alignItems="center"
+          justify="center"
+        >
+          {props.arrow === "prev" ? (
+            <span className={styles.PrevNextArrow}>«</span>
+          ) : (
+            " "
+          )}
+        </Grid>
+
+        <Grid
+          item
+          xs={11}
+          container
+          direction="column"
+          alignItems="center"
+          justify="center"
+        >
+          <LazyLoad once height={100} offset={300}>
+            <img
+              height="100"
+              src={
+                article?._embedded?.["wp:featuredmedia"]?.[0]?.media_details
```

Comment:
Any chance of one of these ending up as a null pointer?

Reply:
I think it's good because if the optional chaining `?.` results in a missing property it'll return `undefined`, in which case the `||` (below) will assign it the `AppbarLogo`